### DISCUSSION TITLE
HMIP-SWDO unreach

### DISCUSSION THREAD

**Discussion by mrt1976** [OpenCCU/OpenCCU#1912] [🇩🇪 Fragen&Antworten] [answered]
Hallo

Seit dem vorletzten Update der Raspberrymatic stehen alle 3 HMIP-SWDO auf unreach und geben keinen Status mehr.
Habt ihr eine Idee. Batterien sind erneuert worden. Mit dem letzten Update leider keine Verbesserung

**Comment by Baxxy13** [comment, marked as answer]
Der nächste Schritt wäre den Sensor (strikt nach Anleitung) lokal werksresetten. 
Dann sollte er sich wieder mit der Zentrale verbinden.

Hilft das nicht bleibt nur die eQ-3 Holzhammermethode...
aus der Zentrale löschen
Sensor lokal werksresetten
neu anlernen

**Comment by mrt1976** [comment]
Danke dir...Geht wieder

**Reply by Baxxy13** [reply]
Sehr gut. Was hat's gebracht, die Holzhammermethode oder der einfache lokale Reset?

**Comment by mrt1976** [comment]
Der lokale Reset 👍
